src/Sdk/Models/AddAliasRequest.cs

```diff
@@ -0,0 +1,12 @@
+namespace Passwordless.Net.Models;
+
+public class AddAliasRequest
+{
+    public string UserId { get; set; }

```

Comment:
If `UserId` is required we should enforce it through a constructor. Also, just guessing that `Aliases` must include at least one alias on this request so we should probably require that through the constructor. Maybe even 2 constructors, one that takes just a single string and we create the `HashSet<string>` for them since that might be a common scenario.

Reply:
@justindbaur  check again